tools/fuzzer/topology.c

```diff
@@ -0,0 +1,1215 @@
+/*
```

Comment:
I didn't look into details. But by quick look it looks like a lot is duplicated from testbench, cannot we make  it less duplicated?

Reply:
@jajanusz the main difference between the testbench and the fuzzer parser is the IPC. I suppose I could make the IPC sending parts conditional and re-use the same topology.c file. Let me give it a try.